mitodl/open-discussions
# Added embedly link preview content to index

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1811 

#### What's this PR do?
Embedly loads some preview content when we enter some URL for a link posts. This PR indexes that content as part of the post to make it more searchable and more relatable to other posts (the indexed preview content comes from the newly added `Post.preview_text` model field)

#### How should this be manually tested?
- Apply migration
- Create a link post for some URL and submit after Embedly loads
- Go to the search page and search for some term that appears in the body of the embedly preview, but does not appear in your post title. You should see this link post in the results.

These posts should also show up in the related post sidebar if the preview content has similar text to whatever post you're viewing.

#### Any background context you want to provide?

#### Screenshots (if appropriate)
![ss 2019-02-13 at 15 31 04](https://user-images.githubusercontent.com/14932219/52742441-bbe58000-2fa5-11e9-83e6-fbba865324a5.png)
![ss 2019-02-13 at 15 31 25](https://user-images.githubusercontent.com/14932219/52742416-ac663700-2fa5-11e9-96e0-64ac33e6be91.png)
![ss 2019-02-13 at 15 32 03](https://user-images.githubusercontent.com/14932219/52742417-ac663700-2fa5-11e9-8daf-05c09f3e47af.png)
![ss 2019-02-13 at 15 32 29](https://user-images.githubusercontent.com/14932219/52742418-ac663700-2fa5-11e9-863c-100637c872b3.png)
